tests/charts/test_statsd.py

```diff
@@ -51,6 +52,17 @@ def test_should_add_volume_and_volume_mount_when_exist_extra_mappings(self):
             "subPath": "mappings.yml",
         } in jmespath.search("spec.template.spec.containers[0].volumeMounts", docs[0])
 
+    @parameterized.expand([(8,), (None,)])
```

Comment:
Why isn't statsd also getting coverage for the global setting?

Reply:
My fault! Fixed